app/forms/validators.py

```diff
@@ -232,11 +232,11 @@ def __call__(self, form, field):
 
         try:
             if hasattr(form, "day"):
-                datetime.strptime(form.data, "%Y-%m-%d")
+                datetime.strptime(form.data, "%Y-%m-%d").replace(tzinfo=timezone.utc)
```

Comment:
It might be worth making a helper method for this as it is done in a lot of places. Thoughts?

Reply:
It is used five times in runner files and three times in tests, I would like to know what @ajmaddaford thinks about this.

Reply:
I can see like 9 usages in runner outside tests.